bundles/persistence/org.openhab.persistence.jdbc/java/org/openhab/persistence/jdbc/internal/JdbcConfiguration.java

```diff
@@ -85,13 +85,14 @@ public void updateConfig(Map<Object, Object> config) {
         String url = (String) configuration.get("url");
         Properties parsedURL = StringUtilsExt.parseJdbcURL(url);
 
-        if (user == null || StringUtils.isBlank(user)) {
+        if (user.equalsIgnoreCase("false") || StringUtils.isBlank(user)) {
```

Comment:
@watou, I will change these points and this: https://github.com/openhab/openhab/pull/4550#discussion-diff-71987438 and create a new PR.